packages/paypal-commerce-integration/src/create-paypal-commerce-integration-service.ts

```diff
@@ -13,10 +13,10 @@ import {
 const createPayPalCommerceIntegrationService = (
     paymentIntegrationService: PaymentIntegrationService,
 ) => {
-    const { getHost } = paymentIntegrationService.getState();
+    const { getHost, getSiteLink } = paymentIntegrationService.getState();
 
     return new PayPalCommerceIntegrationService(
-        createFormPoster(),
+        createFormPoster({ host: getSiteLink() }),
```

Comment:
Can getHost be used here instead of getSiteLink?

Reply:
https://github.com/bigcommerce/checkout-sdk-js/pull/2742#discussion_r1867685384